### ISSUE TITLE
Issue with comma-spacing rule

### ISSUE THREAD

**Issue by SimeonC** [babel/babel-eslint#515]
Originally posted in: https://github.com/eslint/eslint/issues/9225

**Tell us about your environment**

* **ESLint Version:**
4.6.1
* **Node Version:**
8.2.1
* **npm Version:**
5.3.0

**What parser (default, Babel-ESLint, etc.) are you using?**
babel-eslint 7.2.3

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
parser: babel-eslint
extends: airbnb
env:
  browser : true
rules:
  comma-dangle: [2, never]
  max-len:
    - 2
    - 120
    - 2
    - { ignoreUrls: true, ignoreComments: false }
  react/no-find-dom-node: 0
  arrow-body-style: 0
  no-mixed-operators: 0
  no-shadow: [2, { allow: [err, error] }]
  react/prefer-stateless-function: [error, { ignorePureComponents: true }]

  no-console: 2
  no-underscore-dangle: 0
  guard-for-in: 0
  no-param-reassign: [2, { props: false }]

  react/forbid-prop-types: [2, { ignore: ['intl', 'location', 'match', 'history'] }]
  react/prop-types: [2, { ignore: ['intl', 'location', 'match', 'history'] }]

  jsx-a11y/no-static-element-interactions: 0
  jsx-a11y/tabindex-no-positive: 0
  jsx-a11y/img-has-alt: 0
  jsx-a11y/label-has-for: 0

  # Custom rules from ./eslint-rules
  decorator-order: [2, { last: ['inject', 'observer'] }]
```

</details>

**What did you do? Please include the actual source code causing the issue.**

```js


fetch(`https://maps.googleapis.com/maps/api/distancematrix/json?${
  qs.stringify({
    origins: '',
    destination: `place_id:${12312312}`,
    mode: 'walking',
    language: 'en',
    key: config.googleMapsKey
  })
  }`)
.then(response => console.log('distancematrix test', response));
const test = `${1234},${123}`;

```

**What did you expect to happen?**
No Errors to occur.

**What actually happened? Please include the actual, raw output from ESLint.**
```const test = `${1234},${123}`;```
Gives error: `error  A space is required after ','  comma-spacing`

**Comment by SimeonC** [comment]
Currently to fix this I can change it to:

```
const query = qs.stringify({
    origins: '',
    destination: `place_id:${12312312}`,
    mode: 'walking',
    language: 'en',
    key: config.googleMapsKey
  });
fetch(`https://maps.googleapis.com/maps/api/distancematrix/json?${query}`)
.then(response => console.log('distancematrix test', response));
const test = `${1234},${123}`;
```

**Comment by kaicataldo** [comment]
Thank you for the issue. Now that [`@babel/eslint-parser` has been released](https://babeljs.io/blog/2020/07/13/the-state-of-babel-eslint), we are making this repository read-only. If this is a change you would still like to advocate for, please reopen this in the [babel/babel monorepo](https://github.com/babel/babel).
